core/src/main/java/org/apache/carbondata/core/datastore/page/encoding/ColumnPageEncoderMeta.java

```diff
@@ -132,6 +132,10 @@ private void writeMinMax(DataOutput out) throws IOException {
       out.writeDouble((Double) getMaxValue());
       out.writeDouble((Double) getMinValue());
       out.writeDouble(0d); // unique value is obsoleted, maintain for compatibility
+    } else if (dataType == DataTypes.FLOAT) {
+      out.writeDouble((Float) getMaxValue());
```

Comment:
Why not written in Float? Please written in float instead of double

Reply:
used writeFloat instead of writeDouble